covidX/settings.py

```diff
@@ -183,3 +184,4 @@
     "SCHEMA_OUTPUT": "schema.json",
     "SCHEMA_INDENT": 2,
 }
+ALGOLIA = {"APPLICATION_ID": "YourApplicationID", "API_KEY": "YourAdminAPIKey"}
```

Comment:
@premalrupnur write an `os.getenv` to get `ALGOLIA_APPLICATION_ID` and `ALGOLIA_API_KEY` ;
from env vars. then you can do:
`ALGOLIA = {"APPLICATION_ID": ALGOLIA_APPLICATION_ID, "API_KEY": ALGOLIA_API_KEY}`

Reply:
ok will do

Reply:
I have to replace the "ALGOLIA_APPLICATION_ID" with the value returned after using getenv method ,right

Reply:
Yes, ALGOLIA_APPLICATION_ID is a const